src/pages/documentation/adobe-target/event-reference.md

```diff
@@ -0,0 +1,200 @@
+# Target event reference
```

Comment:
As this is the first reference to Target, should this be `Adobe Target event reference`?